### PR TITLE
feat: pause unpause UX

### PR DESCRIPTION
**Pull request by keita-determined**
## Ticket

[ET-293]

## Description

Support pause/unpause in flat run table
> In context menu, if you run is part of a search it should be disabled. In bulk actions dropdown, we send request unmodified, and we need to interpret error where trial is part of a multi trial and raise it as part of a toast.

from [ET-293]
## Test Plan

- Pause/Unpause should work in the context menu and bulk action

## Checklist

- [ ] Changes have been manually QA'd
- [ ] New features have been approved by the corresponding PM
- [ ] User-facing API changes have the "User-facing API Change" label
- [ ] Release notes have been added as a separate file under `docs/release-notes/`
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses have been included for new code which was copied and/or modified from any external code

[ET-293]: https://hpe-aiatscale.atlassian.net/browse/ET-293?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DET-123]: https://hpe-aiatscale.atlassian.net/browse/DET-123?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/utils/flatRun.test.ts [hunk 1]
```diff
@@ -214,5 +215,29 @@ describe('Flat Run Utilities', () => {
         expect(canActionFlatRun(FlatRunAction.Move, bothArchivedFlatRun)).toBeFalsy();
       });
     });
+
+    describe('Pause Action', () => {
```
Comment (ashtonG) [inline webui/react/src/utils/flatRun.test.ts:219]: remember to check the branch where the run is multitrial

Comment (keita-determined) [reply webui/react/src/utils/flatRun.test.ts:219]: added

Comment (ashtonG) [reply webui/react/src/utils/flatRun.test.ts:219]: you're only checking (trial state is  pausable AND experiment is not multitrial) and (trial state is not pausable AND experiment is multitrial). You need to be sure to test (trial state is pausable AND experiment is multitrial) and (trial state is not pausable AND experiment is not multitrial).

**Review by johnkim-det — COMMENTED** [review]
functionality looks good!

one minor issue is that the toasts for bulk actions say "1 runs" instead of "1 run" -- not introduced by this PR but should be a quick fix to use the `pluralizer` util in `submitBatchAction` in `TableActionBar` and `FlatRunActionButton`

also think patch coverage will probably hit 80% by adding assertions that mocked `pauseRuns` and `resumeRuns` are called (see `killRuns` test case example).

File: webui/react/src/utils/flatRun.test.ts (hunk 1 above)
Comment (keita-determined) [reply webui/react/src/utils/flatRun.test.ts:219]: this should be fixed

**Comment by keita-determined** [comment]
> functionality looks good!
> 
> one minor issue is that the toasts for bulk actions say "1 runs" instead of "1 run" -- not introduced by this PR but should be a quick fix to use the `pluralizer` util in `submitBatchAction` in `TableActionBar` and `FlatRunActionButton`
> 
> also think patch coverage will probably hit 80% by adding assertions that mocked `pauseRuns` and `resumeRuns` are called (see `killRuns` test case example).

fixed

**Review by ashtonG — APPROVED** [review]

**Pull request merged**